README.md

```diff
@@ -1,67 +1,51 @@
+Visit [Foxglove Docs](https://docs.foxglove.dev/) to get started.
 
-## Introduction
+## Getting started
 
-See [Foxglove Schemas documentation](https://docs.foxglove.dev/docs/visualization/message-schemas/introduction).
+### Python
 
-The [schemas](./schemas) folder contains type definitions generated for ROS 1, ROS 2, Protobuf, JSON Schema, TypeScript, and OMG IDL.
+| Package                     | Version                                                                                                                         | Description                                                                                                                                  |
+| --------------------------- | ------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------- |
+| foxglove-websocket          | [![pypi version](https://shields.io/pypi/v/foxglove-websocket)](https://pypi.org/project/foxglove-websocket/)                   | Python implementation of the WebSocket protocol<br/>Source: [foxglove/ws-protocol](https://github.com/foxglove/ws-protocol/tree/main/python) |
+| foxglove-schemas-protobuf   | [![pypi version](https://shields.io/pypi/v/foxglove-schemas-protobuf)](https://pypi.org/project/foxglove-schemas-protobuf/)     | Compiled protocol buffers                                                                                                                    |
+| foxglove-schemas-flatbuffer | [![pypi version](https://shields.io/pypi/v/foxglove-schemas-flatbuffer)](https://pypi.org/project/foxglove-schemas-flatbuffer/) | Compiled flatbuffers                                                                                                                         |
 
-These schemas can be used in [MCAP](https://github.com/foxglove/mcap) files or [Foxglove WebSocket](https://github.com/foxglove/ws-protocol) servers to take advantage of Foxglove's visualizations.
+### C++
 
-## License
+Note: C++ implementation currently lives in [foxglove/ws-protocol](https://github.com/foxglove/ws-protocol/tree/main/cpp).
```

Comment:
Its not really the "c++ implementation" of the SDK and this might confuse folks - you can say that the c++ version is coming soon but for now you can use the ws-protocol repo for live connection and the mcap library for recording?

Reply:
updated description